modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/application.ftl

```diff
@@ -1,15 +1,9 @@
 package ${configYAML.apiPackagePath}.internal.jaxrs.application;
 
-<#if freeMarkerTool.isUseJavax(configYAML)>
-	import javax.annotation.Generated;
+<#assign javaEePrefix = freeMarkerTool.getJavaEePrefix(configYAML) />
 
-	import javax.ws.rs.core.Application;
-
-<#else>
-	import jakarta.annotation.Generated;
-
-	import jakarta.ws.rs.core.Application;
-</#if>
+import ${javaEePrefix}.annotation.Generated;
```

Comment:
Should this be called javaEEPrefix & getJavaEEPrefix? what do you think @4lejandrito @magjed4289

Reply:
can you update it then?

Reply:
Apply it on top of my changes.